RFC/src/RFC-0300_DAN.md

```diff
@@ -1 +1,205 @@
+
+### Validator Nodes
+
+[Validator Node]s form the basis of the second layer DAN. All actions on this network take place by interacting with VN's. Some examples of actions
+that VNs will facilitate are:
+* Issuing a [Digital Asset],
+* querying the state of [Digital Asset] and its constituent [tokens],
+* issuing and instruction to change the state of a [Digital Asset] or [tokens].
+
+VNs will also perform archival functions for the assets they manage. The lifetime of these archives and the fee structure for this function is
+still being discussed.
+
+#### Registration
+VNs register themselves on the [Base Layer] using a special [transaction] type. The registration [transaction] type
+requires the spending of a certain minimum amount of [Tari coin], the ([RegistrationCollateral]), that has a time-lock on the
+output for a minimum amount of time ([RegistrationTerm]) as well as some metadata, such as the VNs public key and a generated Node ID. The Node ID is generated
+during registration to prevent mining of VN public keys that can be used to manipulate routing on the DAN. The blinding factor for the Registration transaction is the private key
+that the VN node will use to sign every instruction that it executes for the duration of its [RegistrationTerm].
+
+Once a VNs [RegistrationTerm] has expired so will this specific VN registration. The UTXO timelock will have elapsed so the [RegistrationCollateral] can be reclaimed and a new VN registration
+need to be performed. This automatic registration expiry will ensure that the VN registry stays up to date with active VN registrations and inactive registrations will naturally be removed.
+
+Requiring nodes to register themselves serves two purposes:
+* Makes VN Sybil attacks expensive,
+* Provides an authoritative "central-but-not-centralised" registry of validator nodes from the base layer.
+
+#### Execution of instructions
+VNs are expected to manage the state of digital assets on behalf of digital asset issuers. They receive fees as reward
+for doing this.
+* Digital assets consist of an initial state plus a set of state transition rules. These rules are set by the Tari
+  protocol, but will usually provide parameters that must be specified by the [Asset Issuer].
+* The set of VNs that participate in managing state of a specific DA is called a [Committee]. A committee is selected during the asset
+issuance process and membership of the committee can be updated at [Checkpoint]s.
+* It is the VNs responsibility to ensure that every state change in a digital asset conforms to the contract's rules.
+* VNs accept digital asset [Instructions] from clients and peers. [Instructions] allow for creating, updating, and
+  expiring digital assets on the DAN.
+* VNs provide additional collateral when accepting an offer to manage an asset, which is stored in a multi-signature
+  contract on the base layer. This collateral can be taken from the VN if it is proven that the VN engaged in
```

Comment:
Mimblewimble cannot store contracts on the base layer

Reply:
reworded